plotly/offline/offline.py

```diff
@@ -129,6 +129,8 @@ def init_notebook_mode(connected=False):
         script_inject = (
             ''
             '<script>'
+            'window.is_loaded = false;'
+            'setTimeout(function(){{window.is_loaded=true}},100);'
```

Comment:
I'd rather we *didn't* inject something into the global namespace to do this (`is_loaded`). Also, JS variables are typically in `camelCase` --> `isLoaded`.

If we *do* need to do this, can it be something like `plotlyIsLoaded`?